addons/binding/org.openhab.binding.groheondus/README.md

```diff
@@ -86,6 +86,6 @@ Number:Temperature Temperature_Sense_Guard "Temperature [%.1f %unit%]" {channel=
 
 String Name_Sense "Temperature [%.1f %unit%]" {channel="groheondus:sense:groheondus:appliance:444e8400-e29b-11d4-a716-446655440000:name"}
 Number:Temperature Temperature_Sense "Temperature [%.1f %unit%]" {channel="groheondus:sense:groheondus:appliance:444e8400-e29b-11d4-a716-446655440000:temperature"}
-Number Humidity_Sense "Humidity [%.1f %unit%]" {channel="groheondus:sense:groheondus:appliance:444e8400-e29b-11d4-a716-446655440000:humidity"}
+Number Humidity_Sense "Humidity [%.1f]" {channel="groheondus:sense:groheondus:appliance:444e8400-e29b-11d4-a716-446655440000:humidity"}
```

Comment:
You did not change it to `Number:Dimensionless` but removed the `%unit%` placeholder instead which was fine for this item because we use it as default pattern for the `humidity` channel type too.

Reply:
Ok, I may not getting this comment, it's late and it was a hard day, so sorry if this question is stupid and the answer is obvious :]

Should %unit% be re-added here or not?

Reply:
Thanks for your update. There are no stupid questions :wink:.

> Should %unit% be re-added here or not?

Yes, it should.

Reply:
@FlorianSW One final open item and we are ready to merge. Can you take care of it?